tests/modules/sightings/resources/test_create_sighting.py

```diff
@@ -21,7 +21,7 @@ def test_create_failures(flask_app_client, test_root, researcher_1):
     assert not response.json['success']
 
     # has encounters, zero assetReferences, but fails on bad (missing) context value
-    data_in = {'encounters': [{'taxonomy': {'id': '0000000'}}]}
+    data_in = {'encounters': [{}]}
```

Comment:
why not? Surely this interface hasn't changed... yet?

Reply:
No, but that bit isn't used yet. If you don't have a matching taxonomy (and there isn't a taxonomy '0000000' in my local) then the EDM gets mad. Didn't seem relevant to what I was testing to add default taxonomies and validate them.

When we start getting more into validating encounter and sighting info I think we'll need to push more test data like this into the EDM database as a first step. Actually sort of looking at that for IA config lists elsewhere.